NuRadioReco/modules/io/coreas/readCoREASShower.py

```diff
@@ -98,12 +113,14 @@ def run(self):
             # create sim shower, no core is set since no external detector description is given
             sim_shower = coreas.make_sim_shower(corsika)
```

Comment:
Did you check if this still works as expected with the changes in #688  ?

Reply:
No. But I not that should be addressed in #688

Reply:
Agreed, but I was just interested to know if we need to keep this in our mind when wrapping up #688

Reply:
yes definitely!